server/portal/settings/settings.py

```diff
@@ -34,6 +34,9 @@ def gettext(s): return s  # noqa:E731
     os.path.join(BASE_DIR, 'fixtures'),
 ]
 
+# this can be set to just '/' if we're not using core portal to create cms sessions
+LOGOUT_REDIRECT_URL='/cms/logout/'
```

Comment:
Should this be a setting configurable in `settings_custom.py`?

Reply:
I think it could be updated in `settings_custom.py` (maybe with some changes?) or in `settings_local.py`